resource_dispatcher/src/server.py

```diff
@@ -43,10 +43,14 @@ def sync(self, parent, children):
                 )
                 return {"status": {}, "children": []}
 
-            desired_secret_count = len(glob.glob(f"{folder}/*.yaml"))
+            desired_secrets_count = len(glob.glob(f"{folder}/secrets/*.yaml"))
+            desired_svc_accounts_count = len(glob.glob(f"{folder}/service-accounts/*.yaml"))
             desired_resources = []
             desired_status = {
-                "resources-ready": str(len(children["Secret.v1"]) == desired_secret_count)
+                "resources-ready": str(
+                    len(children["Secret.v1"]) == desired_secrets_count
+                    and len(children["ServiceAccount.v1"]) == desired_svc_accounts_count
+                )
```

Comment:
I know where this comes from, but it also feels superficial that the code's test for if everything is ready is just that the count is the same.  What if we've changed the secrets or SAs asked for?  What if `children[secret]=myOldSecret` and `desired_resources[secret]=myNewSecret`?  

I'm fine with leaving this, but we should add a comment describing this limitation and saying it is just a rough attempt